src/views/partials/milestone-block.handlebars

```diff
@@ -0,0 +1,23 @@
+<div class="card">
+	<header class="card-header">
+		<p class="card-header-title">
+			{{title}}
+		</p>
+		<a class="card-header-icon" aria-label="more options">
+		<span class="icon">
+			<i class="fas fa-angle-down" aria-hidden="true"></i>
+		</span>
+		</a>
+	</header>
+	<div class="card-content">
+		<div class="content">
+			<p>Priority: {{priority}}</p>
```

Comment:
We could also return the data sorted by priority from the backend, or implement a front-end sorting script.